dpop/DPoP-client-app/src/main/java/org/wso2/dpop/client/GeneratePublicKeyPair.java

```diff
@@ -0,0 +1,62 @@
+package org.wso2.dpop.client;
+
+import com.nimbusds.jose.jwk.Curve;
+import com.nimbusds.jose.jwk.ECKey;
+import com.nimbusds.jose.jwk.JWK;
+import com.nimbusds.jose.jwk.RSAKey;
+
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.security.InvalidAlgorithmParameterException;
+import java.security.Key;
+import java.security.KeyPair;
+import java.security.KeyPairGenerator;
+import java.security.NoSuchAlgorithmException;
+import java.security.interfaces.ECPublicKey;
+import java.security.interfaces.RSAPublicKey;
+import java.util.Scanner;
+
```

Comment:
Add class level comments. fix in all places

Reply:
Done